app/scripts/sys-view-model.js

```diff
@@ -39,7 +57,13 @@ window.SysViewModel = (function () {
             return 'label label-' + compileStatusToLabelClassMap[self.compileStatus()];
         });
         self.compileBtnEnable = ko.pureComputed(function () {
-            return !(self.compileStatus() === 'Waiting' || self.compileStatus() === 'Compiling');
+            var status = (self.compileStatus() === 'Waiting' || self.compileStatus() === 'Compiling');
```

Comment:
minor quirk: `status` can be renamed to `ready` and set to the negation of the expression on the right, instead of negating twice below.